Zend/tests/bug47109.phpt

```diff
@@ -7,4 +7,4 @@ $a->{"a"."b"};
 --EXPECTF--
 Warning: Undefined variable $a in %s on line %d
 
-Warning: Trying to get property 'ab' of non-object in %s on line %d
+Warning: Access to property 'ab' on unknown in %s on line %d
```

Comment:
We could restore the original message in this case. Although, in this case it would be nice to adapt the similar message for invalid method calls  as well.

Reply:
This should say `on null`. It should be fine to just treat `IS_UNDEF` as `IS_NULL` for this purpose.

Reply:
Will change it then!